example/iris

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-java -Xmx2G -cp ../brushfire-scalding/target/scala-2.10/brushfire-scalding-0.6.0-SNAPSHOT-jar-with-dependencies.jar \
+java -Xmx2G -cp ../brushfire-scalding/target/scala-2.10/brushfire-scalding-0.6.4-SNAPSHOT-jar-with-dependencies.jar \
```

Comment:
This doesn't actually work for me on either my laptop or runway for some reason, but I think unrelated to this PR.